app/views/teacher_training_adviser/steps/_previous_teacher_id.html.erb

```diff
@@ -1,5 +1,23 @@
-<% @page_title = "What is your previous teacher reference number?" %>
-
-<%= f.govuk_fieldset legend: { text: "What is your previous teacher reference number?", tag: "h1" } do %>
+<%= f.govuk_fieldset legend: { text: "What is your teacher reference number (TRN)?", tag: "h1" } do %>
   <%= f.govuk_text_field :teacher_id, width: 20 %>
 <% end %>
+
+<details class="govuk-details" data-module="govuk-details">
+  <summary class="govuk-details__summary">
+    <span class="govuk-details__summary-text">
+      How do I find my teacher reference number (TRN)?
+    </span>
+  </summary>
+  <div class="govuk-details__text">
+    Your TRN is sometimes on your payslip, pension statement or teacher training records.<br>
```

Comment:
You can find your TRN on:
- a payslip or pension statement
- teacher training records
Or
- You can use the [Find a lost TRN](https://find-a-lost-trn.education.gov.uk/start) service